client/service/src/config.rs

```diff
@@ -123,6 +123,10 @@ pub struct Configuration<G, E = NoExtension> {
 	pub tracing_targets: Option<String>,
 	/// Tracing receiver
 	pub tracing_receiver: sc_tracing::TracingReceiver,
+	/// The size of the instances cache.
+	///
+	/// The default value is 2.
```

Comment:
Yet `default()` below sets it to zero

Reply:
Nice catch! One more reason why I don't like to set the default value in cli.